server/api/auth/login/ldap.post.ts

```diff
@@ -32,21 +34,22 @@ function getSSLConfig(): false | { rejectUnauthorized: boolean; ca: string | Buf
   }
 
   return {
-    rejectUnauthorized: config.ldap_ssl_insecure !== "true",
+    rejectUnauthorized: !config.ldap_ssl_insecure,
```

Comment:
this should probably be `!ldapSSL`, if we just used this change then you'd run into the issue of config.ldap_ssl_insecure being a string value of whatever so the '!' in front would not behave how you would expect.